tools/nl/validator/utils.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import datetime
+import json
+import os
+
+PLACE_PREF = [
+    'State',
+    'County',
+    'Country',
+    'AdministrativeArea1',
+    'AdministrativeArea2',
+    'EurostatNUTS2',
+    'EurostatNUTS3',
+    'EurostatNUTS1',
+]
+
+
+#
+# Loads the bootstrap stuff into `bootstrap`.
```

Comment:
nit: this comment feels very meta without being very informative :)

Reply:
Reworded